bin/MadGraph5_aMCatNLO/cards/WW_helicity/ggWW_LL/ggWW_LL_proc_card.dat

```diff
@@ -0,0 +1,8 @@
+import model loop_sm
+
+define l- = mu- e- ta-
+define l+ = mu+ e+ ta+
+
+generate g g > w+{0} w-{0} $$ h [noborn=QCD]
```

Comment:
what's the reason to use "$$ h", since it will remove interference between z/h, have you checked the effect after removing the Higgs contribution